package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
Seems like we're adding new functionality, so this should be a minor version bump
```suggestion
  "version": "2.1.0",
```